phpunit.xml

```diff
@@ -35,7 +35,7 @@
         <env name="DB_DATABASE" value="tenancy"/>
         <env name="TENANCY_SYSTEM_CONNECTION_NAME" value="mysql"/>
         <env name="QUEUE_DRIVER" value="sync"/>
-        <env name="APP_ENV" value="production"/>
+        <env name="APP_ENV" value="testing"/>
```

Comment:
This should be "production", because we need to test our code as it is running in production.

Reply:
NP, This is not needed now that the tests have been updated.